mlflow/pipelines/pipeline.py

```diff
@@ -231,7 +228,7 @@ class Pipeline:
     """
 
     @experimental
-    def __new__(cls, profile: str) -> RegressionPipeline:
```

Comment:
Can we keep this now and extend it to a Union later, once we have the classification pipeline introduced?

Reply:
Absolutely, updated